scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -4124,6 +4124,16 @@ def run(self):
                 self.__report_container_metrics,
             )
         )
+
+        server = self._global_config.scalyr_server
+        server = server.replace("https://agent.", "https://www.")
+        server = server.replace("https://log.", "https://www.")
```

Comment:
This is common code with the `status -v` output right?  We should make a common method in `util.py` that returns the URL for all logs from this `serverHost` given the `scalyr_server` and `server` value.

I don't think this is quite right though, which probably means we didn't update the `status -v` when we should have.  For example, in the K8s instructions for EU, we tell the customer to set the scalyr server to `https://upload.eu.scalyr.com`.  This code here would mean that they would be visiting `https://upload.eu.scalyr.com` to view their events in the Scalyr UI.

I would suggest that anything mapping to (upload,agent,log,app).eu.scalyr.com gets mapped to `https://eu.scalyr.com`.  Anything mapping to (upload,agent,log,app).scalyr.com gets mapped to `https://www.scalyr.com`.  Otherwise, we just leave the server host unchanged.